src/vsr/replica.zig

```diff
@@ -5298,15 +5387,16 @@ pub fn ReplicaType(
                     .commit_max = self.commit_max,
                     .view = self.view,
                     .log_view = self.log_view,
-                    .headers = self.create_view_change_headers(),
+                    .headers = &self.view_headers,
```

Comment:
No need to regenerate the headers here, they are already set up:
- If `log_view == view`, this is an SV — we set these to our suffix during `primary_start_view_as_the_new_primary`.
- If `log_view < view`, this is a DVC — we set these when we transitioned to `status=view_change`.